<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deposit</class>
 <widget class="QDialog" name="deposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_amount">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Deposit amount</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_amount">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_term">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Term (mounth)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_placement">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_rate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Interest rate (%)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_rate">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_tax_rate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Tax rate (%)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tax_rate">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_periodicity">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Periodicity of payments</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Interest capitalization</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_replenishment">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Replenishment list</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_withdrawals">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>List of partial withdrawals</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_list_replenishment">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>291</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_list_withdrawals">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>291</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_calculate">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>320</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_calculation_results">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>361</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_interest_charges">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>361</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_result_1">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>361</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_tax_amount">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>361</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_result_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>361</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_amount_end_term">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>361</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_result_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>361</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
